homeassistant/package_constraints.txt

```diff
@@ -215,3 +215,8 @@ rpds-py==0.26.0
 
 # Constraint num2words to 0.5.14 as 0.5.15 and 0.5.16 were removed from PyPI
 num2words==0.5.14
+
+# pymodbus does not follow SemVer, and it keeps getting
+# downgraded or upgraded by custom components
+# This ensures all use the same version
+pymodbus==3.11.1
```

Comment:
would it make sense to still use a range? Because the .z version changes should have no API changes in them, and I don't want to "punish" custom components for missing 3.11.1 and being built for 3.11.0

Reply:
Oh right, never mind